pkg/csi/cinder/nodeserver.go

```diff
@@ -411,6 +411,24 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		}
 	}
 
+	// Try expanding the volume if it's created from a snapshot or another volume
```

Comment:
can we link the issue number as comment so it's easier to know why we add such action here?

Reply:
Good point, added :)